src/atomate2/qchem/flows/core.py

```diff
@@ -0,0 +1,169 @@
+"""Core QChem Flows"""
+
+from copy import deepcopy
+from dataclasses import dataclass, field
+from pathlib import Path
+
+from jobflow import Flow, Maker
+from pymatgen.core.structure import Molecule
+
+from atomate2.qchem.jobs.base import BaseQChemMaker
+from atomate2.qchem.jobs.core import (
+    OptMaker,
+    TransitionStateMaker,
+    FreqMaker,
+)
+
+
+# from atomate2.qchem.schemas.calculation import VaspObject
+# from atomate2.vasp.sets.core import HSEBSSetGenerator, NonSCFSetGenerator
+
+__all__ = [
+    "FrequencyFlatteningOptimizeMaker",
+    "FrequencyFlatteningTransitionStateMaker",
+]
+
+
+@dataclass
+class FrequencyFlatteningOptimizeMaker(Maker):
+
+    #need to incorporate iterations and perturb geometry
+    """
+    Maker to iteratively optimize given structure and flatten imaginary frequencies to ensure that
+    the resulting structure is a true minima and not a saddle point.
+
+    Parameters
+    ----------
+    name : str
+        Name of the flows produced by this maker.
+    freq_maker : .BaseQChemMaker
+        Maker to generate the frequency calculation.
+    opt_maker : .BaseQChemMaker
+        Maker to generate the optimization calculation.
+    """
+
+    name: str = "frequency flattening structure optimization"
+    max_iterations=10
+    max_molecule_perturb_scale=0.3
+    linked=True
+    freq_before_opt=False
+    perturb_geometry=False
+    mode=None
+    scale=1.0
+    max_errors=20
+
+    opt_maker: BaseQChemMaker = field(default_factory=OptMaker)
+    freq_maker: BaseQChemMaker = field(default_factory=FreqMaker)
+
+    def make(self, molecule: Molecule, prev_qchem_dir: str or Path or None = None, mode=mode, scale=scale):
```

Comment:
You can't specify types like that in type hinting. You need to use the `|` character. You also can't use `mode` or `scale` since they are dataclass variables. Instead you should remove these parameters and use `self.mode` in the function.
```suggestion
    def make(self, molecule: Molecule, prev_qchem_dir: str | Path | None = None):
```